packages/@okta/vuepress-site/docs/reference/api/devices/index.md

```diff
@@ -1182,9 +1198,33 @@ The following diagram shows the state object for a Device:
         "meid": null,
         "udid": "36A56558-1793-5B3A-8362-ECBAA14EDD2D",
         "sid": null,
-        "tpmPublicKeyHash":null,
-        "registered":true,
-        "secureHardwarePresent":false
+        "tpmPublicKeyHash": null,
+        "registered": true,
+        "secureHardwarePresent": false,
+        "diskEncryptionType": null
+    }
+}
+```
+
+```json
```

Comment:
Do we need 2 examples if the only difference is that the second one has `integrityJailbreak` while the first one does not? Can we clump it to 1?

Reply:
Thoughts? @brentschaus-okta